rtb/piano_di_progetto_ver0.2.0.typ

```diff
@@ -229,3 +236,75 @@ Gli analisti evidenziano le difficoltà e i dubbi relativa alla comprensione dei
 I verificatori suggeriscono infine di incaricare più di una persona alla verifica dei documenti ad alto contenuto, quali possono essere le Norme di Progetto e il Piano di Progetto, al fine di garantire una verifica ottimale.
 
 
+#pagebreak()
+
+=== Sprint 2
+- *Inizio*: 28/11/2024
+- *Fine*: 13/12/2024
+- *Rischi attesi*: #link(<RO-2>)[RO-2], #link(<RO-3>)[RO-3], #link(<RT-1>)[RT-1]
+
+==== Attività dello sprint
+//Quello che è stato fatto o quello che è da fare ?
+Le attività che il _team_ si è prefissato di portare a termine durante la durata del secondo _sprint_ sono le seguenti:
+- Redazione verbale esterno ed interno del 27/11/2024.
+- Creare una prima versione del simulatore dati.
+  - È stato deciso di dedicare la prima settimana dello _spint_ allo studio ed all'approfondimento delle tecnologie: #rifGlossario("Docker"), #rifGlossario("Apache Kafka") ed #rifGlossario("Apache Nifi")
+  - Mentre di dedicare la seconda settimana all'implementazione concreta del simulatore, utilizzando le tecnologie studiate.
```

Comment:
- eviterei di continuare la frase in un elenco puntato, piuttosto direi "La seconda settimana sarà riservata a ..."
- non c'è solo la parte di simulazione ma anche il salvataggio/popolamento dei dati in db